### PR TITLE
Update translations.coffee

### PR DESCRIPTION
**Pull request by nokiacheung**

### CODE REVIEW

File: addon/locales/zh-tw/translations.coffee [hunk 1]
```diff
@@ -53,15 +53,15 @@ I18nTranslationsZhTw =
...
-    "title": "Rate us!"
-    "message": "It won’t take more than a minute and helps to promote our app. Thanks for your support :)"
-    "cancel_button_label": "No, Thanks"
-    "later_button_label": "Remind Me Later"
-    "rate_button_label": "Sure"
-    "yes_button_label": "Yes!"
-    "no_button_label": "Not really"
-    "app_rate_prompt_title": "Thank you, We appreciate your kindness. Give GoodCity a rating?"
-    "feedback_prompt_title": "Mind giving us some feedback?"
+    "title": "Rate us!" "給我們評分吧!"
```
Comment (Darshan-Chauhan) [inline addon/locales/zh-tw/translations.coffee]: @nokiacheung , you need to remove every english word there is after ":"

for example in "title": "Rate us!" "給我們評分吧!" , **remove Rate us**

File: addon/locales/zh-tw/translations.coffee [hunk 2]
```diff
@@ -53,15 +53,15 @@ I18nTranslationsZhTw =
...
+    "message": "It won’t take more than a minute and helps to promote our app. Thanks for your support :)" "不需一分鐘便能幫助推廣我們的應用程式。感謝您的支持:)＂
```
Comment (Darshan-Chauhan) [inline addon/locales/zh-tw/translations.coffee]: @nokiacheung , same for all other translations. :)

**Comment by Darshan-Chauhan** [comment]
@nokiacheung , Thanks :) 👍

**Comment by namrataukirde** [comment]
👍

**Pull request merged**